spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/:user_id/posts/:id' do
+    it 'works! (now write some real specs)' do
+      get '/users/1/posts/1'
+      expect(response).to have_http_status(200)
+      expect(response).to render_template('posts/show')
+      expect(response.body).to include('Post 1 from User 1')
+    end
+  end
+end
```

Comment:
- [ ] Please ensure that index and show actions for  Post controllers are tested, you can create two blocks for index and show and add your tests inside 👍🏿 
````ruby
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
````

- Make sure that for each action you check:
If response status was correct.
If a correct template was rendered.
If the response body includes correct placeholder text.